packages/metal-state/test/validators.js

```diff
@@ -211,10 +211,12 @@ describe('validators', function() {
 		const NAME = 'name';
 		const PARENT_COMPONENT_NAME = 'parentComponent';
 
+		/* eslint-disable */
 		const ERROR_MESSAGE = `Error: Warning: Invalid state passed to '${NAME}'. ` +
```

Comment:
Just curious, why does eslint complain about this? Is it an anti pattern to use the new string templates with concatenation?